Closes #2134 add general option signif_digits (#2212)
pharmaverse/admiral: vignettes/lab_grading.Rmd

due to floating point issues. To solve this issue {admiral} used the `signif()` function on
both side of the equation, the number of significant digits used to compare is passed into the 
function `derive_var_atoxgr_dir()` via the argument `signif_dig`. Please see documentation of the 

> function for more details and also blog: [How admiral handles floating points](https://pharmaverse.github.io/blog/posts/2023-10-30_floating_point/floating_point.html).

Reviewer: ```suggestion
function for more details and the blogpost [How admiral handles floating points](https://pharmaverse.github.io/blog/posts/2023-10-30_floating_point/floating_point.html) for more context.
```

Reviewer: Looks good to me! 👍 👍 👍 👍

> function for more details and the blogpost [How admiral handles floating points](https://pharmaverse.github.io/blog/posts/2023-10-30_floating_point/floating_point.html) for more context.

Author: ```suggestion
function for more details and the blog post [How admiral handles floating points](https://pharmaverse.github.io/blog/posts/2023-10-30_floating_point/floating_point.html) for more context.
```
